src/vs/editor/browser/services/treeSitter/treeSitterParserService.ts

```diff
@@ -264,9 +288,11 @@ export class TreeSitterTextModelService extends Disposable implements ITreeSitte
 	readonly _serviceBrand: undefined;
 	private _init!: Promise<boolean>;
 	private _textModelTreeSitters: DisposableMap<ITextModel, TextModelTreeSitter> = this._register(new DisposableMap());
-	private _registeredLanguages: DisposableMap<string, IDisposable> = this._register(new DisposableMap());
+	private readonly _registeredLanguages: Map<string, string> = new Map();
```

Comment:
(feel free to ignore)

```suggestion
	private readonly _registeredLanguages = new Map<string, string>();
```